pandas/tests/frame/test_api.py

```diff
@@ -164,6 +164,11 @@ def test_iteritems(self):
         for k, v in compat.iteritems(df):
             assert type(v) == Series
 
+    def test_items(self):
+        df = DataFrame([[1, 2, 3], [4, 5, 6]], columns=['a', 'a', 'b'])
+        for k, v in df.items():
+            assert type(v) == Series
```

Comment:
I just cargo-culted the test immediately above this that checks `compat.iteritems(df)` :smiling_imp: